ckanext/canada/tables/admin_aircraft.yaml

```diff
@@ -0,0 +1,330 @@
+      fr: Description de la raison du voyage, en anglais, en service commandé visé. La description doit être brève (une ligne dans la mesure du possible) et donner une indication générale aux utilisateurs du but du voyage. Il faut éviter d’utiliser des acronymes.
+    obligation: Mandatory
+    excel_required: true
+    form_required: true
+    format_type: Free text
+    validation: This field must not be empty
+    visible_to_public: Yes
+    occurrence: Single
+    datastore_type: text
+    excel_column_width: 30
+    form_attrs:
+      size: 60
+
+  # 3.4
+  - datastore_id: purpose_fr
+    label:
+      en: Purpose of the trip (French)
+      fr: But du voyage (français)
+    description:
+      en: A description, in French, of the reason for the government trip in question. The description should be succinct (one line if possible) and provide users with a general sense of the trip's purpose. Use of acronyms should be avoided, in French
+      fr: Description de la raison du voyage, en français, en service commandé visé. La description doit être brève (une ligne dans la mesure du possible) et donner une indication générale aux utilisateurs du but du voyage. Il faut éviter d’utiliser des acronymes.
+    obligation: Mandatory
+    excel_required: true
+    form_required: true
+    format_type: Free Text
+    validation: This field must not be empty
+    visible_to_public: Yes
+    occurrence: Single
+    datastore_type: text
+    excel_column_width: 30
+    form_attrs:
+      size: 60
+
+  # 3.5
+  - datastore_id: start_date
+    label:
+      en: Travel start date
+      fr: Date du début du voyage
+    description:
+      en: The date the travel started.
+      fr: Date du début du voyage.
+    obligation: Mandatory
+    excel_required: true
+    form_required: true
+    format_type: Date (Please format the data as YYYY-MM-DD)
+    validation: This field must not be empty
+    visible_to_public: Yes
+    occurrence: Single
+    datastore_type: date
+    form_snippet: scheming/form_snippets/date.html
+
+# 3.6
+  - datastore_id: end_date
+    label:
+      en: Travel end date
+      fr: Date de fin du voyage
+    description:
+      en: The date the travel ended (can be the same as travel start date).
+      fr: Date de fin du voyage (peut être la même date que la date du début du voyage).
+    obligation: Mandatory
+    excel_required: true
+    form_required: true
+    format_type: Date (Please format the data as YYYY-MM-DD)
+    validation: This field must not be empty
+    visible_to_public: Yes
+    occurrence: Single
+    datastore_type: date
+    form_snippet: scheming/form_snippets/date.html
+
+  # 3.7
+  - datastore_id: locations_en
+    label:
+      en: Location(s) visited (English)
+      fr: Endroit(s) visité (anglais)
+    description:
+      en: To include names of all places visited to conduct government business during the trip in question, in English.
```

Comment:
... separated by semicolons (;)

Reply:
Is the order of place names significant? Do we want to validate that the same number of places are provided in english and french?